docs/sources/flow/get-started/install/puppet.md

```diff
@@ -0,0 +1,105 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/get-started/install/puppet/
+canonical: https://www.puppet.com/
```

Comment:
I think you need a different canonical here.
`https://grafana.com/docs/agent/latest/flow/get-started/install/puppet/`